alien-cas.sh

```diff
@@ -18,5 +26,19 @@ MODULEFILE="$MODULEDIR/$PKGNAME"
 mkdir -p "$MODULEDIR"
 alibuild-generate-module > "$MODULEFILE"
 cat >> "$MODULEFILE" <<EOF
-setenv X509_CERT_DIR \$::env(BASEDIR)/AliEn-CAs/\$version/globus/share/certificates
+try {
+  exec openssl verify \$::env(HOME)/.globus/usercert.pem > /dev/null 2> /dev/null
+} trap CHILDSTATUS {} {
+  # If we couldn't validate the user's cert, either it doesn't exist
+  # or the CERN CA isn't installed as a system cert. In the latter
+  # case, we should redirect to CVMFS or to our certs.
+  if [file isdirectory /cvmfs/alice.cern.ch/etc/grid-security/certificates] {
+    # Prefer current certs from CVMFS, if available. This avoids
+    # failures due to expired certs.
+    setenv X509_CERT_DIR /cvmfs/alice.cern.ch/etc/grid-security/certificates
```

Comment:
Rather than an hardcoded single place, I would use the latest version of this package, so that we can still update the grid by simply bumping something in alidist and no "out of band" procedures are involved.

Reply:
I.e. something like `ls \$::env(BASEDIR)/AliEn-CAs/ | tail -1`...